exercises/scale-generator/canonical-data.json

```diff
@@ -0,0 +1,169 @@
+{
+  "exercise": "scale-generator",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "comments": [
+        "These tests have no interval.",
+        "Chromatic scale is considered the default scale"
+      ],
+      "description": "Chromatic scales",
+      "cases": [
+        {
+          "description": "Chromatic scale",
+          "property": "pitches",
+          "input": {
+            "tonic": "C"
+          },
+          "expected": [ "C", "C#", "D", "D#", "E", "F", "F#", "G", "G#", "A", "A#", "B" ]
+        },
+        {
+          "description": "Another chromatic scale",
+          "property": "pitches",
+          "input": {
+            "tonic": "F"
+          },
+          "expected": [ "F", "Gb", "G", "Ab", "A", "Bb", "B", "C", "Db", "D", "Eb", "E" ]
+        }
+      ]
+    },
+    {
+      "comments": [
+        "These tests all have intervals and are explorations of different",
+        "traversals of the scale."
+      ],
+      "description": "Scales with specified intervals",
+      "cases": [
+        {
+          "description": "Major scale",
+          "property": "pitches",
+          "input": {
+            "tonic": "C",
+            "intervals": "MMmMMMm"
+          },
+          "expected": [ "C", "D", "E", "F", "G", "A", "B" ]
+        },
+        {
+          "description": "Another major scale",
```

Comment:
> would it be helpful to note in the description that this chromatic scale contains flats instead of sharps? It would be more informative than "another"

same, except now "major" and contains a sharp instead of neither